### ISSUE TITLE
one shunt per bank ? Enhancement 

### ISSUE THREAD

**Issue by N1c084** [stuartpittaway/diyBMS-CurrentShunt#11]
Hi,
Any idea to drive (without too much work) 1 shunt per bank?

Maybe the simple way is to have 1 contrôler per bank, but there is an easy way to have one master controler to obtain all data in one Web UI?

Thanks

**Comment by stuartpittaway** [comment]
Technically it's just reading more MODBUS devices (protocol the shunt uses). The controller code would need significant changes to support this as well

**Comment by N1c084** [comment]
yes i know for modbus communication, but i try to find a simple way to handle it, if possible.
the idea of a master controler is also a big code chalange?

**Comment by stuartpittaway** [comment]
Master controller is a much bigger change

**Comment by N1c084** [comment]
Ok.
Please can you globally detail the step required to add multiple modbus
shunt reading and reporting to web Ui ?
I will try some experience to do it

Le sam. 20 août 2022 à 21:10, Stuart Pittaway ***@***.***> a
écrit :
